tests/unit/http/test_mixins.py

```
@@ -978,11 +961,12 @@ async def test_async_col_mx_iterate_handles_api_errors(async_dummy_service):
     ],
 )
 def test_modifieable_resource_mixin(method_name):
+    """Test that ModifiableResourceMixin has the required methods."""
+
     class Service(ModifiableResourceMixin[DummyModel]):  # noqa: WPS431
```

Ruff (0.14.6)

Unused `noqa` directive (unknown: `WPS431`)

Remove unused `noqa` directive

(RUF100)